Ginger/GingerCoreNETUnitTest/GingerCoreNETUnitTest.csproj

```diff
@@ -114,6 +113,12 @@
     <ProjectReference Include="..\GingerRuntime\GingerRuntime.csproj" />
   </ItemGroup>
```

Comment:
Unit tests can be added to cover it, we have a lot of JSON unit tests which Global Variables section can be added and be validate in some of them

Reply:
sure will add it in different PR